install/dc-detect-version.sh

```diff
@@ -9,22 +9,20 @@ fi
 echo "${_group}Initializing Docker Compose ..."
 
 # To support users that are symlinking to docker-compose
-dc_base="$(docker compose version &>/dev/null && echo 'docker compose' || echo 'docker-compose')"
+dc_base="$(docker compose version --short &>/dev/null && echo 'docker compose' || echo '')"
 dc_base_standalone="$(docker-compose version &>/dev/null && echo 'docker-compose' || echo '')"
```

Comment:
Might as well use --short here too 
```suggestion
dc_base_standalone="$(docker-compose version --short &>/dev/null && echo 'docker-compose' || echo '')"
```